resources/views/menus/builder.blade.php

```diff
@@ -277,7 +277,7 @@
              */
             $('.item_actions').on('click', '.delete', function (e) {
                 id = $(e.currentTarget).data('id');
-                $('#delete_form')[0].action = $('#delete_form')[0].action.replace("__id",id);
+                $('#delete_form')[0].action = $('#delete_form')[0].action.replace("/[^\/]*$/",id);
```

Comment:
Not working for me, it doesn't replace anything.
Why not instead use:
```suggestion
                $('#delete_form')[0].action = '{{ route('voyager.menus.item.destroy', ['menu' => $menu->id, 'id' => '']) }}' + id;
```

Reply:
But we need to change the menu id in modal popup form action. 
It's dynamic because there is not a single menu they are so many with different menu id.

Reply:
We do that. It just attaches the dynamic ID to the "static" route `admin/menus/X/item/ + id`